test/RibbonV2SharePrice.test.ts

```diff
@@ -0,0 +1,134 @@
+import { artifacts, ethers, waffle } from 'hardhat';
+import { expect } from 'chai';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address';
+import { MockCozyToken, MockRibbonVaultV2, RibbonV2SharePrice } from '../typechain';
+
+const { deployContract } = waffle;
+const { parseUnits } = ethers.utils;
+
+describe('RibbonV2SharePrice', function () {
```

Comment:
Are ribbon vaults designed similarly to yearn vaults in that we can create protection markets for other ribbon vaults using the same trigger contract with different constructor parameters? If so, I'd recommend setting up this test file similarly to `YearnCrvTwoTokens.test.ts`, which runs the test suite for multiple vaults